modules/openxr/action_map/openxr_action_map.cpp

```diff
@@ -354,6 +354,31 @@ void OpenXRActionMap::create_default_action_sets() {
 	profile->add_new_binding(haptic, "/user/hand/left/output/haptic,/user/hand/right/output/haptic");
 	add_interaction_profile(profile);
 
+	// Create our Pico 4 Ultra controller profile.
```

Comment:
I'm hesitant to adding this to the default action map. We really should clean up the action map to only have entries we must have.

That said, until it's updated by PICO, without the fallback to the pico 4 (non ultra) interaction profile we're kind of stuck.